File: lib/task_manager.rb
class TaskManager
  def initialize
    @tasks = []
  end

  def add_task(task)
    @tasks << task
  end

  def remove_task(task)
    @tasks.delete(task)
  end

  def view_tasks
    @tasks
  end
end

if __FILE__ == $PROGRAM_NAME
  manager = TaskManager.new
  manager.add_task("Task 1")
  manager.add_task("Task 2")
  manager.add_task("Task 3")
  puts "Tasks: #{manager.view_tasks}"
end

File: task_manager.py
class TaskManager:
    def __init__(self):
        self.tasks = []

    def add_task(self, task):
        self.tasks.append(task)

    def remove_task(self, task):
        self.tasks.remove(task)

    def view_tasks(self):
        return self.tasks

if __name__ == "__main__":
    manager = TaskManager()
    manager.add_task("Task 1")
    manager.add_task("Task 2")
    manager.add_task("Task 3")
    print("Tasks:", manager.view_tasks())
